### ISSUE TITLE
Revive SemVer projects

### ISSUE THREAD

**Issue by alexandrtovmach** [mojombo/semver#573]
Hello community :wave:  
I just got maintainers rights and hope to make this project active, again. Of course, the development of specifications never stopped, and we have stable updates from the maintainers and the community:

![image](https://user-images.githubusercontent.com/28801003/84289295-fc79ed00-ab4a-11ea-87a2-6166d15bf0b6.png)

Specifications shouldn't be changed twice a day, because it's stable monolith that using by millions products around the world, and it's totally fine to merge to `master` once in 3 months. But... on other hand we have hundreds issues/PRs, that should be resolved, that's why I'm here :construction_worker_man: :muscle:
I have a plan how to do this:

#### Resolve legacy
- [x] mark each open issue/PR with labels
- [x] ping authors of staled contributions to help with resolving their questions
- [x] resolve critical issues/PRs
- [x] close staled things
- [ ] investigate to the rest of issues/PRs and try to resolve them

#### Updates
- [x] add validation tools to standardize contributions and translations (https://github.com/semver/semver.org/pull/270, https://github.com/semver/semver/pull/578)
- [x] configure CI/CD (GitHub Actions) (https://github.com/semver/semver.org/pull/266, https://github.com/semver/semver.org/pull/270, https://github.com/semver/semver/pull/578)
- [ ] write/update contribution guidelines and community related docs (#421, #410, https://github.com/semver/semver.org/issues/57)
- [x] add process to sync [semver/semver.org](https://github.com/semver/semver.org) with [semver/semver](https://github.com/semver/semver) (#407, https://github.com/semver/semver.org/pull/265)

Any notes or suggestions about this plan really appreciated. Thanks!

**Comment by alexandrtovmach** [comment]
![image](https://user-images.githubusercontent.com/28801003/84525616-97f29580-ace4-11ea-9bac-934721695c72.png)
![image](https://user-images.githubusercontent.com/28801003/84525647-a5a81b00-ace4-11ea-9e4c-062857fb8460.png)

*39 issues and PRs* already resolved 🎉🎉🎉
keep moving forward 🏃

**Comment by alexandrtovmach** [comment]
## 7 days summary
33 issues and PRs resolved 🚀🚀🚀 
![image](https://user-images.githubusercontent.com/28801003/85174672-7febba80-b27e-11ea-8833-17a4a7484a25.png)
![image](https://user-images.githubusercontent.com/28801003/85174634-6ea2ae00-b27e-11ea-9096-22c52dd81ddc.png)

Plan progress:
~resolve critical issues/PRs~ **DONE**
~close staled things~ **DONE**
~add validation tools to standardize contributions and translations~ **DONE**
~configure CI/CD (GitHub Actions)~ **DONE**

I'm really happy to be a part of SemVer community! Let's move on and make versioning better 🏃
Thanks to all contributors, you're the best 🥇

**Comment by robertgiessmann** [comment]
hi @alexandrtovmach , first of all: thanks for the great work in getting this repo maintained! You indicated:

> Any notes or suggestions about this plan really appreciated. Thanks!

I have the following idea to contribute: should SemVer go for "official Web Standard" status by becoming a W3C Recommendation (https://www.w3.org/standards/faq.html#std)? What do you think? 

Best, Robert

**Comment by ljharb** [comment]
Why would the w3c standardize something browsers have no interest in using?

**Comment by robertgiessmann** [comment]
Hi there @ljharb , true, maybe not w3c then? What about the IETF, or others? Do you have someone in mind who would be a suitable standardization body?

**Comment by klehelley** [comment]
Regarding writing an IETF RFC, see issue #253

**Comment by robertgiessmann** [comment]
True, issue #253 it is. Sounds to me like there's no desire of the community (fine if it is like this)?

**Comment by Neustradamus** [comment]
Any progress?
